### PR TITLE
made editableOptions.activate 'select' option to fallback on focus…

### PR DESCRIPTION
**Pull request by Carnid**
… when element cannot be selected

### CODE REVIEW

**Comment by ckosloski** [comment]
Do all the tests still pass?

File: src/js/editable-element/controller.js [hunk 1]
```diff
@@ -416,8 +416,12 @@ angular.module('xeditable').factory('editableController',
           }
           el.focus();
         }
-        if (editableOptions.activate === 'select' && el.select) {
-          el.select();
+        if (editableOptions.activate === 'select') {
```
Comment (ckosloski) [inline src/js/editable-element/controller.js]: Maybe make this an else if since activate can only be one value.

**Comment by Carnid** [comment]
How do you run your tests? Are they manual? I have found the tests themselves in the docs, but you don't seem to have a test runner setup.

**Comment by ckosloski** [comment]
- For testing locally
  - install grunt globally, if you haven't already: `npm install -g grunt-cli`
  - `npm install` in the project
  - `grunt jade` - to create the test files
  - `grunt build` - to build the project files
  - `npm i http-server -g`
  - `http-server`
  - http://127.0.0.1:8080/test/e2e/docs-test.html
  - http://localhost:8000/test/e2e/dev-test.html

**Comment by Carnid** [comment]
All tests pass.

**Comment by ckosloski** [comment]
Can you squash your commits into one?

**Comment by ckosloski** [comment]
Also, try navigating to http://127.0.0.1:8080/ and verify that the examples in the documentation work as you now expect them to work.

**Comment by Carnid** [comment]
Well, to do that, I would need to add a section that highlights the `editableOptions.activate = 'select'` feature, which is not yet documented there. Also, since it's an angular run config, I believe it would need a second ng-app to demonstrate.

Unless I change the whole demo app to use `'select'` instead of `'focus'`.

What would you suggest?

**Comment by Carnid** [comment]
Well, it seems my attempts to squash my two commits went terribly wrong. Want me to do another PR instead?

**Comment by ckosloski** [comment]
Yeah, if it's easier, go ahead and create a new pull request.
For testing, just update the app.js and test the documentation examples locally to make sure they work.  Then revert the app.js changes before you create your new pull request.

**Pull request closed without merging**